ColorListGenerator/Color.swift

```diff
@@ -39,19 +39,21 @@ class Color {
         }
     }
 
-    class func colorFromHexString(value: String) -> NSColor? {
+    class func colorFromHexString(value: String, var alpha: CGFloat = 1.0) -> NSColor? {
         var red: CGFloat = 0.0
         var green: CGFloat = 0.0
         var blue: CGFloat = 0.0
-        var alpha: CGFloat = 1.0
         let characters = Array(value.utf8)
         if 6 <= characters.count {
             red = Color.colorValue(characters[0], characters[1])
             green = Color.colorValue(characters[2], characters[3])
             blue = Color.colorValue(characters[4], characters[5])
         }
-        if 8 <= characters.count {
-            alpha = Color.colorValue(characters[6], characters[7])
+        if 8 == characters.count {
+            alpha = Color.colorValue(characters[0], characters[1])
```

Comment:
Since [This sample](https://github.com/griffin-stewie/clg/blob/268cfbf078d0edd679b59c773482e5ea698e5cd7/samples/sample.json#L3) is using RGBA format, not ARGB, this change is not valid for all the cases.